> Let me know when this is ready and I will review it.

Many thanks, Travis! There are still a lot of failing CI runs, but as far as I see they are all unrelated, for example in [test-long a-f](https://github.com/sagemath/sage/actions/runs/15192409975/job/42728323767?pr=40149#step:9:3980) it is due to a missing shared library for Singular:

```
#31 4376.4   [sagemath_doc_html-none]   [spkg-install]   File "/sage/src/sage/schemes/curves/projective_curve.py", line 144, in <module>
#31 4376.4   [sagemath_doc_html-none]   [spkg-install]     from sage.libs.singular.function import singular_function, lib as singular_lib, get_printlevel, set_printlevel
#31 4376.4   [sagemath_doc_html-none]   [spkg-install]   File "sage/libs/singular/function.pyx", line 98, in init sage.libs.singular.function
#31 4376.4   [sagemath_doc_html-none]   [spkg-install] ImportError: libSingular-4.4.0.so: cannot open shared object file: No such file or directory
#31 4376.4   [sagemath_doc_html-none]   [spkg-install] Error: 'sage --docbuild --all-documents' failed
#31 4376.4   [sagemath_doc_html-none]   [spkg-install] make[5]: *** [Makefile:41: doc-inventory-reference] Error 1
#31 4376.4   [sagemath_doc_html-none]   [spkg-install] make[5]: Target 'doc-html' not remade because of errors.
```

So I set it ready for review!

--- a/src/sage/knots/knotinfo.py
+++ b/src/sage/knots/knotinfo.py
@@ -2029,7 +2029,7 @@ class KnotInfoBase(Enum):
             deprecation(40149, "base_ring is deprecated, use argument ring instead.")
 
         ch = ring.characteristic()
-        integral = ch == 0 and not ring.is_field()
+        integral = (ch == 0) and (not ring.is_field())
         if not self.is_knot():
             # KnotJob calculated results only available for knots
             if ch == 0 and ring.is_field():

--- a/src/sage/knots/knotinfo.py
+++ b/src/sage/knots/knotinfo.py
@@ -1899,14 +1899,14 @@ class KnotInfoBase(Enum):
 
         INPUT:
 
-        - ``var1`` -- (default: ``'q'``) the first variable. Its exponents
+        - ``var1`` -- (default: ``'q'``) the first variable; its exponents
           correspond to the height of Khovanov homology
-        - ``var2`` -- (default: ``'t'``) the second variable. Its exponents
+        - ``var2`` -- (default: ``'t'``) the second variable; its exponents
           correspond to the degree of Khovanov homology
         - ``torsion`` -- (default: ``'T'``) additional variable to indicate
           the torsion of the integral homology group corresponding to the
-          monomial. Monomials without it correspond to torsion free ``ring``
-          modules.  If it appears its exponents stands for the modulus of
+          monomial; monomials without it correspond to torsion free ``ring``
+          modules; if it appears, its exponents stands for the modulus of
           the torsion
         - ``ring`` -- (default: ``ZZ`` for knots and ``QQ`` for proper links)
           the ring for the homology

--- a/src/sage/knots/knotinfo.py
+++ b/src/sage/knots/knotinfo.py
@@ -1972,7 +1972,7 @@ class KnotInfoBase(Enum):
             sage: K8kr == K8ko                                         # optional database_knotinfo
             False
 
-        Caution:
+        Caution::
 
             sage: Kk2 = K.khovanov_polynomial(ring=GF(2))
             Traceback (most recent call last):

